test/rng_testsuite/extra_methods_tests/engines_methods.pass.cpp

```diff
@@ -0,0 +1,263 @@
+                (oneapi::dpl::minstd_rand::increment != MINSTD_C) || (oneapi::dpl::minstd_rand::modulus != MINSTD_M) ||
+                (engine.min() != MINSTD_MIN) || (engine.max() != MINSTD_MAX));
+    }
+    if constexpr (std::is_same<Engine, oneapi::dpl::minstd_rand_vec<N>>::value)
+    {
+        return ((oneapi::dpl::minstd_rand_vec<N>::multiplier != MINSTD_A) ||
+                (oneapi::dpl::minstd_rand_vec<N>::increment != MINSTD_C) ||
+                (oneapi::dpl::minstd_rand_vec<N>::modulus != MINSTD_M) || (engine.min() != MINSTD_MIN) ||
+                (engine.max() != MINSTD_MAX));
+    }
+    if constexpr (std::is_same<Engine, oneapi::dpl::ranlux24_base>::value)
+    {
+        return ((oneapi::dpl::ranlux24_base::word_size != RANLUX24_BASE_W) ||
+                (oneapi::dpl::ranlux24_base::short_lag != RANLUX24_BASE_S) ||
+                (oneapi::dpl::ranlux24_base::long_lag != RANLUX24_BASE_R) || (engine.min() != RANLUX24_BASE_MIN) ||
+                (engine.max() != RANLUX24_BASE_MAX));
+    }
+    if constexpr (std::is_same<Engine, oneapi::dpl::ranlux24_base_vec<N>>::value)
+    {
+        return ((oneapi::dpl::ranlux24_base_vec<N>::word_size != RANLUX24_BASE_W) ||
+                (oneapi::dpl::ranlux24_base_vec<N>::short_lag != RANLUX24_BASE_S) ||
+                (oneapi::dpl::ranlux24_base_vec<N>::long_lag != RANLUX24_BASE_R) ||
+                (engine.min() != RANLUX24_BASE_MIN) || (engine.max() != RANLUX24_BASE_MAX));
+    }
+    if constexpr (std::is_same<Engine, oneapi::dpl::ranlux24>::value)
+    {
+        return ((oneapi::dpl::ranlux24::block_size != RANLUX24_P) ||
+                (oneapi::dpl::ranlux24::used_block != RANLUX24_R) || (engine.min() != RANLUX24_BASE_MIN) ||
+                (engine.max() != RANLUX24_BASE_MAX));
+    }
+    return 0;
+}
+
+template <class Engine>
+bool
+test()
+{
+    using result_type = typename Engine::scalar_type;
+
+    // Catch asynchronous exceptions
+    auto exception_handler = [](sycl::exception_list exceptions) {
+        for (std::exception_ptr const& e : exceptions)
+        {
+            try
+            {
+                std::rethrow_exception(e);
+            }
+            catch (sycl::exception const& e)
+            {
+                std::cout << "Caught asynchronous SYCL exception during generation:\n" << e.what() << std::endl;
+            }
+        }
+    };
+
+    sycl::queue queue(sycl::default_selector{}, exception_handler);
+    int sum = 0;
+
+    // Memory allocation
+    std::vector<std::int32_t> dpstd_res(N_GEN);
+    constexpr std::int32_t num_elems =
+        oneapi::dpl::internal::type_traits_t<typename Engine::result_type>::num_elems == 0
+            ? 1
+            : oneapi::dpl::internal::type_traits_t<typename Engine::result_type>::num_elems;
+
+    // Random number generation
+    {
+        sycl::buffer<std::int32_t, 1> dpstd_buffer(dpstd_res.data(), dpstd_res.size());

```

Comment:
`, 1` is extra. `1` is a default value, you can write just `sycl::buffer<std::int32_t> ...`

Reply:
removed